### PR TITLE
refactor: extend docs

### PR DESCRIPTION
**Pull request by deemp**
- include link to pattern docs
- show how to exclude a file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,7 +171,7 @@ to `./dist` whenever you make a change in `./index.js`.
 
 Do the following steps to make a new release:
 
-1.  Add entries to [`CHANGELOG.md`](./CHANGELOG.md) for all changes since the
+1. Add entries to [`CHANGELOG.md`](./CHANGELOG.md) for all changes since the
     last release.
-2.  Manually create a GitHub release on
+2. Manually create a GitHub release on
```
Comment (cdepillabout) [inline README.md]: Can you revert this?

File: action.yml [hunk 2]
```diff
@@ -4,8 +4,18 @@ inputs:
   pattern:
     required: false
     description: >
```
Comment (cdepillabout) [inline action.yml]: I think you might need to change this to:

```suggestion
    description: |
```

File: README.md [hunk 3]
```diff
@@ -142,13 +146,13 @@ versions, but newer (or slightly older) versions may work as well:
 Next, clone this repo:
 
 ```console
-$ git clone git@github.com:haskell-actions/run-fourmolu.git
```
Comment (cdepillabout) [inline README.md]: Could you revert these three similar changes?

**Comment by cdepillabout** [comment]
Thanks, this is a nice improvement!

**Comment by deemp** [comment]
Done. Didn't notice a formatter run.

**Pull request merged**